openlibrary/templates/merge/authors.html

```diff
@@ -90,13 +90,19 @@ <h1>$_("Merge Authors")</h1>
                         <p>${ macros.TruncateString(a.bio, 250) }</p>
                     $else:
                         <p>No description.</p>
-                    <ul>
-                    $for doc in top.docs:
-                        <li><a href="$doc['key']" target="new" title="$_('Open in a new window')">$doc['title']</a> <span class="smaller">$ungettext('%(count)s edition', '%(count)s editions', doc['edition_count'], count=doc['edition_count']),
-                        $if doc.get('first_publish_year'):
-                            <span title="$_('First published in')">$doc['first_publish_year']</span>
-                        </span></li>
-                    </ul>
+                        <ul>
+                            $for doc in top.docs:
+                                $if is_rtl_lang:
+                                    <li><a href="$doc['key']" target="new" title="$_('Open in a new window')" dir="rtl">$doc['title']</a> <span class="smaller" dir="rtl">$ungettext('%(count)s edition', '%(count)s editions', doc['edition_count'], count=doc['edition_count']),
```

Comment:
Maybe something like this can be done? (Idk if this would be acceptable or not, so please use your own judgement. :))
```suggestion
                            $for doc in top.docs:
                                $ title_rtl_dir = ' dir="rtl"' if is_rtl(doc['title']) else ''
                                <li><a href="$doc['key']" target="new" title="$_('Open in a new window')"$title_rtl_dir>$doc['title']</a> <span class="smaller">$ungettext('%(count)s edition', '%(count)s editions', doc['edition_count'], count=doc['edition_count']),
```